compass/sign/RemoteSignatureSource.java

```diff
@@ -21,8 +22,10 @@
  */
 public class RemoteSignatureSource extends SignatureSource {
   private static final Logger log = LoggerFactory.getLogger(RemoteSignatureSource.class);
+  private static final String defaultCacheTtl = "5";
```

Comment:
```suggestion
  public static final String DEFAULT_CACHE_TTL = "5";
```

Constants are snake case according to java conventions